### DISCUSSION TITLE
Why is Two factor authentication enabled when it is disabled.

### DISCUSSION THREAD

**Discussion by Joofthan** [community/community#33160] [Other Feature Feedback, Questions, & Ideas]
I never enabled Two factor authentication. But i got this message on login.  Email Code was required. I do not always have access to my Email-Account. I don‘t want second factor. 

„If you'd like to require verification on every sign in, consider enabling [two-factor authentication](https://docs.github.com/articles/configuring-two-factor-authentication) on your account.“

My Profiles Settings Say:
„Two factor authentication is not enabled yet.“

Why is Two factor required/enabled if it is disabled.

**Comment by hansiemithun** [comment]
This is a jobless jack feature, when it is not enabled, it should not be. Why is it by default enabled and troubling users?
